spec/features/users_can_diagnose_their_issue_spec.rb

```diff
@@ -26,6 +26,7 @@
       .with('first')
       .and_return(
         Question.new(
+          'id' => 'fun',
```

Comment:
So that the stub matches reality, shouldn't `id` be `first` in this case?

Reply:
Yup